test/integration/fake_upstream.h

```diff
@@ -56,10 +56,20 @@ class FakeStream : public Http::StreamDecoder,
   const Http::HeaderMap& headers() { return *headers_; }
   void setAddServedByHeader(bool add_header) { add_served_by_header_ = add_header; }
   const Http::HeaderMapPtr& trailers() { return trailers_; }
-  void waitForHeadersComplete();
-  void waitForData(Event::Dispatcher& client_dispatcher, uint64_t body_length);
-  void waitForEndStream(Event::Dispatcher& client_dispatcher);
-  void waitForReset();
+  ABSL_MUST_USE_RESULT
+  testing::AssertionResult
+  waitForHeadersComplete(std::chrono::milliseconds timeout = std::chrono::milliseconds(10000));
```

Comment:
Consider replacing `std::chrono::milliseconds(10000)` with a static constant.

Also I think it would be easier to read these 3-line declarations if they had a blank line between them.

Reply:
Done (and the rest of these too)